.github/workflows/refapp-3x-login.yml

```diff
@@ -13,5 +18,5 @@ jobs:
           node-version: '12.x'
       - name: Installing dependencies
         run: npm install
-      - name: Run patient registration workflow tests
+      - name: Run login workflow tests
```

Comment:
```suggestion
      - name: Run user settings workflow tests
```
?

Reply:
No. This is the workflow file of the login. There was a typo, and I fixed it.